phpcs.xml

```diff
@@ -21,10 +23,31 @@
     <!-- PHP Rules -->
     <rule ref="PHPCompatibility" />
     <config name="testVersion" value="5.2-"/>
+    <rule ref="Squiz.PHP.DisallowMultipleAssignments.Found">
+        <severity>0</severity>
+    </rule>
 
     <!-- WordPress Rules -->
     <rule ref="WordPress">
         <exclude name="WordPress.VIP"/>
+        <!-- In the API, we process the requests. Nonce verification is to be done
+            at the form processing level prior to calling our API. -->
+        <exclude name="WordPress.CSRF.NonceVerification.NoNonceVerification"/>
+        <exclude name="WordPress.PHP.StrictInArray.MissingTrueStrict"/>
+        <!-- We have functions that can receive more arguments than declared
+            in the function's parameter list. -->
+        <exclude name="Squiz.Commenting.FunctionComment.ExtraParamComment"/>
+        <!-- Not every file needs a file header comment, e.g. Views -->
```

Comment:
I'd suggest that, for consistency, and for the sake of not turning this whole check off for the whole project, that views do have a file-level DocBlock. Here's an [example](https://github.com/GaryJones/plugin-boilerplate/blob/bd9ca02455f5fc2f3b57ade0459d672b9472d392/phpunit.xml.dist#L10). Note that one could also add in namespaces, strict_type declarations and so on, as well as making it clear to anyone opening the view file out of context, what project it is from and where it is used.